### DISCUSSION TITLE
How can I extend PrimeTween

### DISCUSSION THREAD

**Discussion by hoang000147** [KyryloKuzyk/PrimeTween#41] [FAQ] [answered]
For example, I want to extend PrimeTween with this Jump method: forum.unity.com/threads/1479609/#post-9226566

**Comment by KyryloKuzyk** [comment, marked as answer]
You can place the Jump() right inside the class where you want to use it. Or you can place the Jump() method inside a static class, so you can call it from any place in your codebase. Please see the [updated post](https://forum.unity.com/threads/primetween-high-performance-animations-and-sequences.1479609/#post-9226566).
